engine/flink/test-util/src/main/scala/pl/touk/nussknacker/engine/flink/test/FlinkMiniClusterHolder.scala

```diff
@@ -53,7 +62,19 @@ class FlinkMiniClusterHolderImpl(flinkMiniCluster: MiniClusterWithClientResource
     flinkMiniCluster.after()
   }
 
-  override def getClusterClient: ClusterClient[_] = flinkMiniCluster.getClusterClient
+  override def cancelJob(jobID: JobID): Unit =
```

Comment:
Let's add `private[test]` for those four methods - let's developers use execution environment